test/orchard/inspect_test.clj

```diff
@@ -319,3 +341,56 @@
              (:path (-> inspector (inspect/down 0)))))
       (is (= '[:a (nth 2) :b :c (nth 73) (find :foo) key class <unknown>]
              (:path (-> inspector (inspect/down 0) (inspect/down 1))))))))
+
+;; Datafy and Nav
+
+(defn- datafy-section [result]
+  (str/replace (first result) #".*Datafy:" "Datafy:"))
+
+(when datafy-supported?
+  (deftest inspect-class-datafy-test
+    (testing "inspecting a class renders the datafy section"
+      (is (= inspect-result-datafy-class
```

Comment:
I don't like the indirection of having to jump to a var which is a huge string anyway

Please let's code it like this: `(= (slurp (io.resource ,,,))) ,,,`

Reply:
I changed the tests to compare data structures, instead of strings. Would that count instead of using files? I think files have a bit of maintenance burden as well. In case the data structures are still too much in this namespace, we could move them to another one. Wdyt?

Reply:
Alright! Looking great.

Let's just inline the var to avoid indirection. Under it's new shape it's definitely readable 👍

Reply:
Done